x-pack/plugins/cases/common/types/api/custom_field/v1.test.ts

```diff
@@ -100,4 +104,30 @@ describe('Custom Fields', () => {
       ).toContain('The value field cannot be an empty string.');
     });
   });
+
+  describe('CaseCustomFieldNumberWithValidationValueRt', () => {
+    const numberCustomFieldValueType = CaseCustomFieldNumberWithValidationValueRt({
+      fieldName: 'value',
+    });
+    it('should decode strings correctly', () => {
+      const query = numberCustomFieldValueType.decode(123);
```

Comment:
I think this is what the test intended 😅 

```suggestion
      const query = numberCustomFieldValueType.decode('123');
```

Reply:
I intended check number, just wrote the wrong test explanation. Rewrite it to 'should decode number correctly'